python/hail/context.py

```diff
@@ -272,13 +273,22 @@ def import_bgen(self, path, tolerance=0.2, sample_file=None, min_partitions=None
         :param reference_genome: Reference genome to use. Default is :class:`~.HailContext.default_reference`.
         :type reference_genome: :class:`.GenomeReference`
 
+        :param contig_recoding: Dict of old contig name to new contig name. The new contig name must be in the reference genome given by ``reference_genome``.
+        :type contig_recoding: Dict of (str or int) to (str or int).
+
         :return: Variant dataset imported from .bgen file.
         :rtype: :class:`.VariantDataset`
         """
 
         rg = reference_genome if reference_genome else self.default_reference
+
+        if contig_recoding:
```

Comment:
if we're casting everything to a string here anyways, can we just have the `contig_recoding` that the function takes be a dict of (strlike, strlike)?